purchase_fop_shipping/views/purchase_order.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="origin_type_purchase_order_tree">
+        <field name="model">purchase.order</field>
+        <field name="type">tree</field>
+        <field name="inherit_id" ref="purchase.purchase_order_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//tree" position="attributes">
+                <attribute name="decoration-danger">fop_reached == False</attribute>
```

Comment:
@sbejaoui Could you check this behaviour as it seems not working on runbot:

![image](https://user-images.githubusercontent.com/19529533/120774173-5163ff80-c522-11eb-94fd-455e3c40f7c8.png)

![image](https://user-images.githubusercontent.com/19529533/120774204-5aed6780-c522-11eb-8fb8-7584ee1ef566.png)